client/component/comment.js

```diff
@@ -213,14 +215,23 @@ function EditComment({comment, big=false, setComment, topLevel, onEditingDone, o
             onEditingDone(comment);
         }
     }
+    const {isMobile} = getDeviceInfo();
+    const {isTextFieldFocused} = useFocus();
 
     return <View>
         {topLevel && <TopBarActionProvider label={action} disabled={!canPost || inProgress} onPress={onPost} />}
         <EditWidgets widgets={commentEditTopWidgets} comment={comment} setComment={setComment} onCancel={onCancel} />
         {big && <EditWidgets widgets={commentEditBottomWidgets} comment={comment} setComment={setComment} onCancel={onCancel} />}
         <TextField value={comment.text} onChange={text => setComment({...comment, text})} 
-            placeholder={placeholder} autoFocus big={big} testID='comment-edit'
+            placeholder={placeholder} big={big} testID='comment-edit'
             placeholderParams={{authorName: getFirstName(author?.name)}} />
+            {(isTextFieldFocused || comment.text.length > 0) && isMobile && (
```

Comment:
As a tiny extra bit of robustness, I might do comment.text?.length - just in case we get called with a comment that has no text.

Reply:
done